### PR TITLE
Update GOCART settings for p8/p81

### PR DESCRIPTION
**Pull request by bbakernoaa**
This PR updates the GOCART settings for p8/p81 and should be done after [PR#795](https://github.com/NOAA-EMC/global-workflow/pull/795) 

It addresses #816 updating the default wet scavenging coefficients when gocart is enabled.

It addresses #814 by fixing a bug in which when running only ATM-AERO simulations the value from parm/config.aero was not read and therefore defaults to zero.

It addresses #817 by updating the dust inputs for the fengsha dust emission scheme.  These updates include a newer soil database (SOILGRIDSv2) and updates for the drag partition and threshold velocity.  It also appropriately scales the emissions by modifying the alpha value found in the dust component configuration file.

It addresses #815.  With @rmontuoro taking his new position the default location of the GOCART emission files needs to be modified so that the aerosol group can modify the input emissions as necessary for development.  

I tested this on HERA using an ATM-AERO only configuration.  

- [x] Bug fix (non-breaking change which fixes an issue)

**How Has This Been Tested?**

- [x] Forecast-only test on Hera
  
**Checklist**

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes need updates to the documentation. I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] New and existing tests pass with my changes
- [ ] Any dependent changes have been merged and published

@zhanglikate @gjfrost

### CODE REVIEW

**Comment by bbakernoaa** [comment]
Addressing #814, @zhanglikate performed a test with this fix and it works properly. Please see the attached presentation paying attention to `v174` (without the fix) and `v175` with the fix. 

https://docs.google.com/presentation/d/1pIu4AIDKL6zfksOkAmeNhAN1WOQfcGl1/edit?usp=sharing&ouid=111407383710090576995&rtpof=true&sd=true

File: parm/chem/ExtData.other [hunk 1]
```diff
@@ -11,12 +11,12 @@ TROPP            'Pa'    Y       N                  -            0.0      1.0
 DU_SRC            NA  N Y - none none du_src     ExtData/Dust/gocart.dust_source.v5a.x1152_y721.nc
 
 # FENGSHA input files. Note: regridding should be N or E
-DU_CLAY          '1'  Y E           -          none none clayfrac   ExtData/Dust/FENGSHA_SOILGRIDS2017_GEFSv12_v1.0.nc
-DU_SAND          '1'  Y E           -          none none sandfrac   ExtData/Dust/FENGSHA_SOILGRIDS2017_GEFSv12_v1.0.nc
-DU_SILT          '1'  Y E           -          none none siltfrac   /dev/null
-DU_SSM           '1'  Y E %y4-%m2-%d2T12:00:00 none none ssm        ExtData/Dust/FENGSHA_SOILGRIDS2017_GEFSv12_v1.0.nc
-DU_RDRAG         '1'  Y E           -          none none drag_part  ExtData/Dust/FENGSHA_SOILGRIDS2017_GEFSv12_v1.0.nc
-DU_UTHRES        '1'  Y E           -          none none uthres     ExtData/Dust/FENGSHA_SOILGRIDS2017_GEFSv12_v1.0.nc
+DU_CLAY           '1'  Y E           -           none none clayfrac    ExtData/dust/FENGSHA_SOILGRIDS2019_GEFSv12_v1.2.nc
```
Comment (WalterKolczynski-NOAA) [inline parm/chem/ExtData.other]: This path changed the uppercase 'D' in Dust into lowercase, but the directory names didn't actually change.

**Review by WalterKolczynski-NOAA — CHANGES_REQUESTED** [review]
I don't think this actually addresses #815 yet, as there are no changes to AERO_INPUTS_DIR in config.aero. I'm copying Rafaelle's directory to a new location under my account; I'll push an update in the morning after it completes.

File: parm/chem/ExtData.other (hunk 1 above)
Comment (bbakernoaa) [reply parm/chem/ExtData.other]: You are right the filenames should have changed here.  

They should be 

```bash
DU_CLAY           '1'  Y E           -           none none clayfrac    ExtData/Dust/FENGSHA_SOILGRIDS2019_GEFSv12_v1.2.nc
DU_SAND           '1'  Y E           -           none none sandfrac    ExtData/Dust/FENGSHA_SOILGRIDS2019_GEFSv12_v1.2.nc
DU_SILT           '1'  Y E           -           none none siltfrac    /dev/null
DU_SSM            '1'  Y E           -           none none ssm         /dev/null:1.0
DU_RDRAG          '1'  Y E %y4-%m2-%d2t12:00:00  none none albedo_drag ExtData/Dust/FENGSHA_Albedo_drag_v1.nc
DU_UTHRES         '1'  Y E           -           none none uthres      ExtData/Dust/randomforestensemble_uthres.nc
```

And the path that you changed the ExtData to point to is not synced with that path that was there. I don't understand why this was modified when it was fine with pointing to Raffaele's directory.  It should continue to point to a location that the aerosol team has access to. 


You changed it from here `/scratch1/RDARCH/rda-arl-gpu/Barry.Baker/emissions/NASA` to `/scratch2/NCEPDEV/ensemble/noscrub/Walter.Kolczynski/global-workflow/gocart_emissions/Dust/`

**Comment by bbakernoaa** [comment]
@WalterKolczynski-NOAA The `AERO_INPUTS_DIR` should point to a location someone in the Aerosol team has access to. 

It should be here `/scratch1/RDARCH/rda-arl-gpu/Barry.Baker/emissions/NASA/ExtData`

Why is it arbitrarily set to your directory instead of the one above?

**Comment by bbakernoaa** [comment]
These has the latest updates.  Raffaele's directory does not

**Comment by WalterKolczynski-NOAA** [comment]
> @WalterKolczynski-NOAA The `AERO_INPUTS_DIR` should point to a location someone in the Aerosol team has access to.
> 
> It should be here `/scratch1/RDARCH/rda-arl-gpu/Barry.Baker/emissions/NASA/ExtData`
> 
> Why is it arbitrarily set to your directory instead of the one above?

When I changed it, it hadn't been changed away from Rafaelle's directory at all. I think there is also value in separating what the aerosol team is working on from what is officially supporting the workflow. I can update the "official" version whenever requested, but it would be less volatile than pointing to an active development copy, particularly since I also have to sync changes to other machines. Other aspects of global workflow (like the fix directory) work the same way.

Really it should be held under the global group account instead of my personal account, but I somehow still don't have access to that on all machines.

**Comment by bbakernoaa** [comment]
@WalterKolczynski-NOAA if it isn't going to be put under the global group account then I don't see why it can't be put under may account space.  This just seems like an arbitrary decision.

**Comment by WalterKolczynski-NOAA** [comment]
> @WalterKolczynski-NOAA if it isn't going to be put under the global group account then I don't see why it can't be put under may account space. This just seems like an arbitrary decision.

@bbakernoaa: @KateFriedman-NOAA has copied the data to `/scratch1/NCEPDEV/global/glopara/data/gocart_emissions` on Hera. When we put the data on WCOSS2, we'll also use glopara there. Not going to bother moving the data on Dell since it only has a month of life left. On Orion there is no glopara, so the current location under me is just as good as one under Kate.

**Review by WalterKolczynski-NOAA — APPROVED** [review]

**Pull request merged**